internal/controlplane/xds_routes.go

```diff
@@ -66,7 +66,7 @@ func buildPomeriumHTTPRoutes(options *config.Options, domain string) []*envoy_co
 	}
 	// if we're handling authentication, add the oauth2 callback url
 	if config.IsAuthenticate(options.Services) && hostMatchesDomain(options.GetAuthenticateURL(), domain) {
-		routes = append(routes, buildControlPlanePathRoute(options.AuthenticateCallbackPath, false))
+		routes = append(routes, buildControlPlanePathRoute(options.AuthenticateCallbackPath, false), buildControlPlanePathRoute("/oauth2/session/frontchannel-logout", false))
```

Comment:
Coudl you add this to the next line to be consistent with the style?

Reply:
removed this line as the route is now under `.pomerium`